src/js/base/module/Editor.js

```diff
@@ -254,8 +254,7 @@ define([
     // native commands(with execCommand), generate function for execCommand
     var commands = ['bold', 'italic', 'underline', 'strikethrough', 'superscript', 'subscript',
                     'justifyLeft', 'justifyCenter', 'justifyRight', 'justifyFull',
-                    'formatBlock', 'removeFormat',
-                    'backColor', 'fontName'];
+                    'formatBlock', 'removeFormat','backColor'];
```

Comment:
Please add a blank before `backColor` for consistent syntax.

Reply:
i added a blank.